src/Discord.Net/Entities/Emotes/IGuildEmote.cs

```diff
@@ -1,7 +1,44 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
 namespace Discord
 {
-    public interface IGuildEmote : IEmote
+    public interface IGuildEmote : IEmote, ISnowflakeEntity, IDeletable
```

Comment:
Now that I think about it, (and while the concretes are classes and not structs for the moment), is there any pressing reason to keep the name for this as `IGuildEmote` and not `IGuildEmoji`? The API docs don't give credence to the existence of an "emote," and while we could use that as a shared interface for `GuildEmote/GuildEmoji` and `Emoji`, I'd like to cut down on one more nuance (Emote v Emoji) that users would need to be aware of that isn't immediately obvious. 

Ideally, it'd flow as `IEmoji` -> `Emoji` -> `GuildEmoji`. I feel like I'm missing something for why we presently have `Emote` and `GuildEmote`. Is the former really necessary for representing user created guild emoji?